### DISCUSSION TITLE
最強iPad！ iPad Pro 2022有望本月發：Apple M2加持

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS3073-C/Python-Community-GS3073-C#478] [General]
大家期待嗎！Ｍ1的ipad pro性能相當強大，但最近用起來覺得掉電很快，希望下一代能改善～
https://news.xfastest.com/apple/119076/%E6%9C%80%E5%BC%B7ipad%EF%BC%81-ipad-pro-2022%E6%9C%89%E6%9C%9B%E6%9C%AC%E6%9C%88%E7%99%BC%EF%BC%9Aapple-m2%E5%8A%A0%E6%8C%81/

**Comment by brian431** [comment]
覺得對於一般用戶續航力的重要性遠大於計算速度。

**Comment by lx3211** [comment]
其實主要客群還是一般用戶居多，用不了那麼高的效能卻會造成那麼高的耗電量，實在會勸退很多人。

**Comment by Chenszuhong** [comment]
我也覺得電量消耗小比較重要，畢竟不是每個人都要用平板做很多運算的工作，大多是拿來做筆記跟看影片

**Comment by 0965256727** [comment]
個人認為這樣會影響到大多客群的使用體驗

**Comment by xCk27x** [comment]
iPad的主要用途還是以輕量的辦公為主
電腦則負責高強度的工作
除非是要接鍵盤當主力使用
不然感覺有點過頭

**Comment by limitx0** [comment]
我用iPad用不到那樣的效能
如果買了感覺會有點浪費
